### ISSUE TITLE
Hi.

### ISSUE THREAD

**Issue by mustafabozdag** [RumenDamyanov/php-sitemap#131]
I want to use laravel-sitemap on laravel 5.4 with autoload, on controller not route

I'm getting this error.

InvalidArgumentException in FileViewFinder.php line 112:
No hint path defined for [sitemap].

And MyCode like this?

     $config = [
            'use_cache' => false,
            'cache_key' => 'Laravel.Sitemap.',
            'cache_duration' => 3600,
            'testing' => true
        ];
			$sitemap = new Sitemap( $config);

  $sitemap->store('sitemapindex','sitemap'); 

Where am i wrong?

php artisan vendor:publish  

This command give this result : Nothing to publish for tag [].

**Comment by dominicmomentum** [comment]
Did you try to delete your the files under your bootstrap/cache folder? (do not delete app.php and autoload.php)

**Comment by mustafabozdag** [comment]
Yes i tried nothing change, alos config:clear, cache:clear everyting i tired.

**Comment by chris-ruskai** [comment]
Did anything come of this? This is also happening to me on Laravel 5.4.
